election-server/src/bin/http.rs

```diff
@@ -264,19 +351,70 @@ async fn get_ballot_submission_by_transaction(
     Ok(Json(result))
 }
 
+/// The response format of [`get_account_weight`]
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+struct AccountWeightResponse {
+    /// The voting weight calculated for the account queried
+    voting_weight:    u64,
```

Comment:
It's a bit unclear from this which weight this is. Is this the weight they would have if election closed now?

Reply:
Looking below, the value seems to be the initial weight. Is that the intent? Please document if so.

Reply:
Yes that is the intent, will change the comment